### PR TITLE
Added cyberlinks data support with translation

### PR DESCRIPTION
**Pull request by litvintech**
Added:

1. Added cyberlinks data support in approve window
2. Added translations in all supported languages

Support already tested with local built extension, also, with integration in-app

### CODE REVIEW

File: src/common/hash/index.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export function truncHashPortion(str, firstCharCount = str.length, endCharCount = 0, dotCount = 3) {
+	var convertedStr="";
+	convertedStr+=str.substring(0, firstCharCount);
+	convertedStr += ".".repeat(dotCount);
```
Comment (webmaster128) [inline src/common/hash/index.ts:4]: Suggestion: use the `…` character instead (https://en.wikipedia.org/wiki/Ellipsis). This is a single charater that is smaller than three separate dots. Then you don't need the `dotCount`.

Comment (litvintech) [reply src/common/hash/index.ts]: @webmaster128 thanks for the response, did it

**Comment by litvintech** [comment]
@Thunnini fixed conflicts with the latest master with cosmwasm support

**Comment by Thunnini** [comment]
Thank you. This will be included in the next release

**Pull request merged**